modules/html-helpers/html-helpers.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * Class _WPLib_Html_Helpers
+ *
+ * Provide an HTML Helper class for WPLib.
+ *
+ * Basically, this class contributes static methods to the WPLib class using magic methods.
+ *
+ */
+class _WPLib_Html_Helpers extends WPLib_Helper_Base {
+
+	/**
+	 *
+	 */
+	static function on_load() {
+
+		/**
+		 * Register this class as a helper for WPLib.
+		 */
+		self::register_helper( __CLASS__, 'WPLib' );
+
+	}
+
+	/**
+	 *  Convenience method for setting HTTP Content Type for outputting plain text.
+	 */
+	static function plain_text() {
```

Comment:
Sure.  I'll remove.  It was just a convenience function I used occasional when debugging.